packages/client/src/components/SafeSettings/components/Assets/RemoveCollection.js

```diff
@@ -0,0 +1,31 @@
+import { Close } from "components/Svg";
+
+const RemoveCollection = ({ name, onCancel, onNext }) => {
+    const nextButtonClasses = ["button flex-1 p-4", "is-link"];
```

Comment:
Just curious on why we join the classes instead of just declare them in the className

Reply:
Copy pasting from other modals 🤷 😆 
Moved these classes to button className directly, and left it only on modal where the class depends on if the form is valid (Add Vault modal)